proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -313,7 +313,13 @@ def build(self):
             initialize=0.1,
             bounds=(1e-8, 1),
             units=pyunits.dimensionless,
-            doc='Bulk concentration based component rejection')
+            doc='Observed solute rejection')
+        self.over_pressure = Var(
+            self.flowsheet().config.time,
+            initialize=0.1,
+            bounds=(1e-8, 100),
```

Comment:
Shouldn't upper bound be 1 if this is a fraction?

Reply:
You could have an over pressure that is over 100%. I obviously put a very loose bound but it could be over 1.

Reply:
Oh, right- I was thinking it would be less than 1 since you're subtracting 1, but I see how it could be >1.